src/Service.Host/client/src/components/Workstations.js

```diff
@@ -76,11 +86,21 @@ function Workstations() {
     return (
         <Page homeUrl={config.appRoot} showAuthUi={false}>
             <Grid container spacing={3}>
-                <Grid size={12}>
+                <Grid size={11}>
                     <Typography variant="h4">Workstation Utility</Typography>
                 </Grid>
+                <Grid size={1}>
+                    <PermissionIndicator
+                        hasPermission={hasPermission}
+                        hasPermissionMessage="You have create/update workstation permissions"
+                        noPermissionMessage="You do not have create/update workstation permissions"
+                    />
+                </Grid>
                 <Grid size={12}>
-                    <CreateButton createUrl="/stores2/work-stations/create" />
+                    <CreateButton
+                        createUrl="/stores2/work-stations/create"
```

Comment:
better to use the create link in the application state result than to hard code here